src/combinatorics/shadows.lean

```diff
@@ -180,7 +180,7 @@ section shadow
       split,
         intro p, exact ⟨B, p, subset.refl _, subset.refl _⟩,
         rintro ⟨A, _, q⟩, rw ← subset.antisymm_iff at q, rwa q,
-    simp, rw @ih (∂𝒜) B, clear ih,
+    simp, rw iterate, rw @ih (∂𝒜) B, clear ih,
```

Comment:
We should try and get rid of this and the other non-terminal simps, I think there are gonna be quite a few